back/src/main/resources/properties/mail.yml

```
@@ -0,0 +1,11 @@
+# local 환경
+spring:
+  mail:
+    host: smtp.gmail.com
+    port: 587
+    username: ${MAIL_USERNAME}
+    password: ${MAIL_PASSWORD}
+    properties:
+      mail.smtp.auth: true
+      mail.smtp.starttls.enable: true
+    default-encoding: UTF-8
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)